src/Resources/views/CRUD/list__action.html.twig

```diff
@@ -9,12 +9,4 @@ file that was distributed with this source code.
 
 #}
 
-{% extends get_admin_template('base_list_field', admin.code) %}
-
-{% block field %}
-    <div class="btn-group">
-        {% for actions in field_description.option('actions') %}
-            {% include actions.template %}
-        {% endfor %}
-    </div>
-{% endblock %}
+{% extends '@SonataAdmin/_layout/'~sonata_config.getOption('layout') ?? 'AdminLTE_v2'~'/CRUD/list__action.html.twig'  %}
```

Comment:
Not sure if I like the idea to proxy EVERY template. This will add immense overhead when rendering a template with many includes.

Reply:
First solution: Use `proxy` templates
- user should set `sonata_admin.options.layout` 
-  templates CAN be call without layout name`@SonataAdmin/template_name.html.twig`

Second solution: Use templates directly
- user should set `sonata_admin.options.layout` 
-  templates MUST be call with layout name`@SonataAdmin/_layout/AdminLTE_v2/template_name.html.twig`

First solution is more flexible and allow major upgrade without BC-break in templates. 
Second solution have better performance.

WDYT? @sonata-project/contributors 

IMO We should keep first solution and replace it only when performance will be to low.

Reply:
The first thing I thought was "Every merge 3.x into master will might be a conflict if a template is changed, and bug fix/feature in 3.x  could be lost during in the merge into master if they are not properly moved to the new template".

Reply:
What if we move this PR to `3.x`. 
`AdminLTE_v2` will be default templates - default layout for Admin 3.x, deprecated in 4.0, removed in 5.0.
`AdminLTE_v3` (and others) will be in `_layout/AdminLTE_v3` - default layout for Admin 4.x (or AdminLTE 4)

Upgrade from Admin 3.x to 4.0 with `AdminLTE_v2` will be possible with some extra config (default will be AdminLTE v3).

Reply:
> The first thing I thought was "Every merge 3.x into master will might be a conflict if a template is changed, and bug fix/feature in 3.x could be lost during in the merge into master if they are not properly moved to the new template".

And every new feature that involves templates will need to have all the possible themes maintained, or every bugfix will need to be applied twice, etc..

Reply:
Should we close this ?